hub/src/server/config-schema.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "serverName": { "type": "string" },
+    "port": { "type": "integer" },
+    "driver": { "type": { "enum":
+                          [ "aws", "azure", "disk", "google-cloud" ] } },
+    "bucket": { "type": "string" },
+    "readURL": { "type": "string" },
+    "pageSize": { "type": "integer"},
+    "requireCorrectHubUrl": { "type": "boolean" },
+    "cacheControl": { "type": "string" },
+    "whitelist": { "oneOf": [
+      {
+        "type": "array",
+        "items": { "type": "string" }
+      },
+      { "type": ["boolean", "null"] }
+    ]},
+    "validHubUrls": { "oneOf": [
+      {
+        "type": "array",
+        "items": { "type": "string" }
+      },
+      { "type": ["boolean", "null"] }
+    ]},
+    "proofsConfig": { "oneOf": [
+      {
+        "type": "object",
+        "properties": {
+          "proofsRequired": { "type": "string"}
+        },
+        "required": [
+          "proofsRequired"
+        ]
+      },
+      { "type": ["boolean", "null"] }
+    ]},
+    "awsCredentials": {
+      "type": "object",
+      "properties": {
+        "sessionToken": { "type": "string"},
+        "accessKeyId": { "type": "string"},
+        "secretAccessKey": { "type": "string"}
+      }
+    },
+    "azCredentials": {
+      "type": "object",
+      "properties": {
+        "accountName": { "type": "string"},
+        "accountKey": { "type": "string"}
+      }
+    },
+    "gcCredentials": {
```

Comment:
For each of these driver schemas (not just `gcCredentials`), do we know which fields are required?

Reply:
None of them are required. You can set almost all of them via provider specific environment variables (i.e., via the aws, azure, or gcp CLIs) or via Gaia environment variables.

Reply:
Gotcha :+1: